### DISCUSSION TITLE
Conda packages for sphinx-needs and sphinx-data-viewer

### DISCUSSION THREAD

**Discussion by melund** [useblocks/sphinx-needs#939] [General]
We will try to use sphinx-needs for our design documents. Our stack relies on the conda package manager, so I plan to create a [conda-forge](https://conda-forge.org/) package for sphinx-needs.

If anyone of the contributors here want to co-maintain the conda package, then let me know. Otherwise, you can also be added later :)

Btw, it seems the `sphinx-data-viewer` package is a bit out of date. The one on pypi is incompatible with `sphinx-needs` unless I tweak the dependencies a bit.

**Comment by melund** [comment]
sphinx-needs is now available as a conda-forge packages for conda:

https://anaconda.org/conda-forge/sphinx-needs

**Comment by danwos** [comment]
Ohh excellent, thanks for creating a conda package.
Unfortunately, I don't have any experience with conda and none of my current projects is using it.
So it would not make much sense for me to co-maintain it.

But for sure, if you have any problems with it in the future and need some help, I would be happy to support you.
